examples/matrix_multiplier/tests/Makefile

```diff
@@ -0,0 +1,85 @@
+TOPLEVEL_LANG ?= verilog
+SIM ?= icarus
+
+PWD=$(shell pwd)
+
+ifeq ($(OS),Msys)
```

Comment:
There have been some changes to how paths are handled, so this needs to be removed.